bridge/client/app/_components/ktb-sequence-tasks-list/ktb-sequence-tasks-list.component.ts

```diff
@@ -30,52 +25,32 @@ export class KtbSequenceTasksListComponent implements OnInit, OnDestroy {
     if (this._tasks !== value) {
       this._tasks = value;
       this.focusLastSequence();
-    }
-  }
-
-  @Input()
-  get stage(): string | undefined {
-    return this._stage;
-  }
-
-  set stage(value: string | undefined) {
-    if (this._stage !== value) {
-      this._stage = value;
-      this.focusLastSequence();
+      this._changeDetectorRef.markForCheck();
     }
   }
 
   @Input()
```

Comment:
You might need to move the `@Input()` to the setter for the change detection `OnPush` to work correctly.